src/lexedata/importer/interleaved.py

```diff
@@ -9,13 +9,19 @@
 import re
 import csv
 import sys
+import logging
+from pathlib import Path
 
 import openpyxl
 
-if __name__ == "__main__":
+# TODO: move this logger part after the argument parser (after merge)
+logger = logging.getLogger(__file__)
+
+
+def import_interleaved(excel: str, forms_path: str):
     comma_or_semicolon = re.compile("[,;]\\W*")
 
-    ws = openpyxl.load_workbook("bantu/bantu.xlsx").active
+    ws = openpyxl.load_workbook(excel).active
```

Comment:
I think it would be nice to just pass the sheet as argument to this function. That also makes testing easier, because we can use mock sheets.